src/plugins/ui_stateful/clipboard.ts

```diff
@@ -260,6 +272,183 @@ export class ClipboardPlugin extends UIPlugin {
     }
   }
 
+  private convertOSClipboardData(text: string): ClipboardData {
```

Comment:
Maybe this should be in the handlers. It seems kinda strange that the clipboard does have to know the internal state of the CellHandler. And in the future we will add more things handled in the OS clipoard, not just simple text cells